src/main/java/pl/tomaszdziurko/jvm_bloggers/blog_posts/NewBlogPostStoringActor.java

```diff
@@ -1,46 +1,57 @@
 package pl.tomaszdziurko.jvm_bloggers.blog_posts;
 
+import static com.google.common.base.MoreObjects.firstNonNull;
 
 import akka.actor.AbstractActor;
 import akka.actor.Props;
 import akka.japi.pf.ReceiveBuilder;
+
+import com.sun.syndication.feed.synd.SyndContent;
 import com.sun.syndication.feed.synd.SyndEntry;
+
+import lombok.experimental.ExtensionMethod;
 import lombok.extern.slf4j.Slf4j;
 import pl.tomaszdziurko.jvm_bloggers.blog_posts.domain.BlogPost;
 import pl.tomaszdziurko.jvm_bloggers.blog_posts.domain.BlogPostRepository;
 import pl.tomaszdziurko.jvm_bloggers.utils.DateTimeUtilities;
 
-import java.time.LocalDateTime;
 import java.util.Date;
-import java.util.Optional;
+
+import org.apache.commons.lang3.StringUtils;
 
 @Slf4j
+@ExtensionMethod({DateTimeUtilities.class, StringUtils.class})
 public class NewBlogPostStoringActor extends AbstractActor {
 
-    private final BlogPostRepository blogPostRepository;
-
     public NewBlogPostStoringActor(BlogPostRepository blogPostRepository) {
```

Comment:
I do not agree. If constructor were designed to be code-free then why it is even available for programmers? It wouldn't be invented if it were not required. The constructor is for initializing the object and here akka actor initialization takes place. We are defining `PartialFunction` here which is an initialization argument for `Actor`. The code is not executed but passed as an argument. See the [Akka](http://doc.akka.io/japi/akka/2.4.2/akka/japi/pf/ReceiveBuilder.html) you will find more examples how to use Akka framework there.